test-app/backend/test/models/shift/shift-api.test.ts

```diff
@@ -0,0 +1,71 @@
+import app from '../../../src/server';
+import request from 'supertest';
+
+const myShift = {
+  start: new Date().getTime(),
+  end: new Date().getTime(),
+  maxVolunteers: 2,
+  title: 'My Lovely Title',
+  description: 'I have a description',
+  eventAdmin: 'Adam Meza',
+};
+
+const noTitleShift = {
+  start: new Date().getTime(),
+  end: new Date().getTime(),
+  maxVolunteers: 2,
+  description: 'I have a description',
+  eventAdmin: 'Adam Meza',
+};
+
+const badDateShift = {
+  start: new Date().getTime() + 2,
+  end: new Date().getTime(),
+  maxVolunteers: 2,
+  title: 'My Lovely Title',
+  description: 'I have a description',
+  eventAdmin: 'Adam Meza',
+};
+
+const badVolShift = {
+  start: new Date().getTime(),
+  end: new Date().getTime(),
+  maxVolunteers: 0,
+  title: 'My Lovely Title',
+  description: 'I have a description',
+  eventAdmin: 'Adam Meza',
+};
+
+test('POST  /api/new-shift', async () => {
```

Comment:
Can you split each `await request(app)` into a new test?

The test description should be like `POST /api/new-shift; Shift fails to create due to bad no title provided`